gulpfile.js

```diff
@@ -1,40 +1,50 @@
-// IMPORTS
-//##########################################################
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
 'use strict';
```

Comment:
I wonder why does jshint not fails on this, global `use strict` should throw an error